compiler/codegen/src/error.rs

```diff
@@ -33,6 +33,7 @@ pub enum CodegenErrorType {
     DuplicateStore(String),
     InvalidMatchCase,
     NotImplementedYet, // RustPython marker for unimplemented features
+    SymbolLookupError { symbol: String },
```

Comment:
Because this is not a user-side Python error but our own RustPython implementation bug, let's not expose it here.